src/quo2/components/avatars/group_avatar.cljs

```diff
@@ -1,32 +1,47 @@
 (ns quo2.components.avatars.group-avatar
   (:require [quo2.components.icon :as icon]
             [quo2.foundations.colors :as colors]
-            [react-native.core :as rn]))
+            [quo2.theme :as theme]
+            [react-native.core :as rn]
+            [react-native.fast-image :as fast-image]))
 
 (def sizes
-  {:icon      {:small  12
-               :medium 16
-               :large  20}
-   :container {:small  20
-               :medium 32
-               :large  48}})
+  {:icon      {:x-small 12
+               :small   16
+               :medium  16
+               :large   20
+               :x-large 32}
+   :container {:x-small 20
+               :small   28
+               :medium  32
+               :large   48
+               :x-large 80}})
 
-;; TODO: this implementation does not support group display picture (can only display default group
-;; icon).
-(defn group-avatar
-  [_]
-  (fn [{:keys [color size]}]
+(defn- group-avatar-internal
+  [_ _]
+  (fn [{:keys [customization-color theme avatar? size]
```

Comment:
The anonymous function overhead seems unnecessary because there's no initialization code outside the renderer.